ciao_contrib/_tools/specextract.py

```diff
@@ -447,7 +453,7 @@ def _event_stats(self,file,colname,args=None):
 
             with new_pfiles_environemt(ardlib=False,copyuser=False):
```

Comment:
```suggestion
            with new_pfiles_environment(ardlib=False,copyuser=False):
```

This suggests we don't have code that tests this section, as it should have crashed and burned

Reply:
fixed, but reviewing the code a bit more, this conditional use case for the function is not called on in the script